store/src/java/com/zimbra/cs/account/ZAttrConfig.java

```diff
@@ -31971,6 +31971,21 @@ public void unsetMobileBlockedDevices() throws com.zimbra.common.service.Service
         getProvisioning().modifyAttrs(this, attrs);
     }
 
+    /**
+     * Blocked mobile device list for ActiveSync/ABQ
+     *
+     * @param attrs existing map to populate, or null to create a new map
+     * @return populated map to pass into Provisioning.modifyAttrs
+     *
+     * @since ZCS 8.9.0
+     */
+    @ZAttr(id=3090)
+    public Map<String,Object> unsetMobileBlockedDevices(Map<String,Object> attrs) {
```

Comment:
unsetMobileBlockedDevices attribute is not related to current fix. Do we need this change?

Reply:
not required for the current change but this is auto generated when we run `ant generate-getters` after making change to zimbra-attrs.xml